.gitignore

```diff
@@ -350,3 +350,4 @@ MigrationBackup/
 .ionide/
 *.sln
 Project EFT.sln
```

Comment:
already in gitignore on line 352, but it was necessary to push sln changes when adding testing portion of project, so not completely stagnant